CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.
+
+# Email and Chat
+
+- Email: [operator-framework](https://groups.google.com/forum/#!forum/operator-framework)  
+
+## Setting up a dev environment
```

Comment:
Marketplace also works on vanilla Kubernetes. Should we at least mention that here? Just that, this is an example environment and vanilla kube and `kubectl` should work in addition to `oc` ?

Reply:
@kevinrizza do we have a doc detailing steps that can be taken to get marketplace-operator running on vanilla Kubernetes (installing olm -> marketplace-operator)? I was thinking about writing something along the lines of:
```
This section provides detailed instructions on how to set up a development environment on an Openshift cluster. To use maketplace-operator in a vanilla Kubernetes cluster instead, follow the instructions [here] (<link-to-k8s-installation-doc>)  and use the `kubectl`/`oc` command. 
```
But I couldn't find any in the docs folder.

Reply:
>But I couldn't find any in the docs folder.

Isn't that what you are writing :-)

This definitely needs an OpenShift vs plain vanilla Kuberenetes sections to call out the differences.

Reply:
We don't have a doc, but the process should be identical except using `kubectl` instead of `oc`. The operator-sdk works with either.